m5stick-nemo.ino

```diff
@@ -2,19 +2,30 @@
 // github.com/n0xa | IG: @4x0nn
 
 // -=-=-=-=-=-=- Uncomment the platform you're building for -=-=-=-=-=-=-
-#define STICK_C_PLUS
-//#define STICK_C_PLUS2
-//#define STICK_C
-//#define CARDPUTER
+// #define STICK_C_PLUS
+// #define STICK_C_PLUS2
+// #define STICK_C
+// #define CARDPUTER
 // -=-=- Uncommenting more than one at a time will result in errors -=-=-
 
-String buildver="2.3.4";
```

Comment:
we no longer need to manage and maintain buildver and bump it everytime. It will be handled automatically by automated build system. We have introduced a pre-processor called `NEMO_VERSION` with default value `dev`. This is the value we will get when someone manually builds and flashes from source code. 

- When the build happens for a branch (i.e. main branch, feature branches), it will use first 7 chars of commit hash.
- When the build is happening for tag, then the tag_name will be replaced automatically. i.e. v0.0.2, etc.

Reply:
Nice :D